src/redux/preview.js

```diff
@@ -148,3 +301,55 @@ module.exports.getRemixes = id => (dispatch => {
         dispatch(module.exports.setRemixes(body));
     });
 });
+
+module.exports.getStudios = id => (dispatch => {
+    dispatch(module.exports.setFetchStatus('studios', module.exports.Status.FETCHING));
+    api({
+        uri: `/projects/${id}/studios?limit=5`
+    }, (err, body) => {
+        if (err) {
+            dispatch(module.exports.setFetchStatus('studios', module.exports.Status.ERROR));
+            dispatch(module.exports.setError(err));
+            return;
+        }
+        if (typeof body === 'undefined') {
+            dispatch(module.exports.setFetchStatus('studios', module.exports.Status.ERROR));
+            dispatch(module.exports.setError('No studios info'));
+            return;
+        }
+        if (body.code === 'NotFound') {
```

Comment:
Probably best caught via `res.statusCode === 404`. `res` is the 3rd argument passed back from api (err, body, res). Ordered that way because most of the time we don't need res, just body.